src/rust/bitbox02/src/lib.rs

```diff
@@ -176,6 +176,12 @@ pub fn reboot() -> ! {
     panic!("reboot called")
 }
 
+pub fn print_stdout(msg: &str) {
```

Comment:
please add `#[cfg(feature = "testing")]` to only activate this in tests, as this should not be used in production firmware.

Reply:
It must have slipped from my eye. Fixing.